### ISSUE TITLE
Add google.protobuf.Duration converter to QTime 

### ISSUE THREAD

**Issue by semlanik** [semlanik/qtprotobuf#84]

**Comment by unC0Rr** [comment]
QTime is just a time part of QDateTime. It makes most sense to convert to std::duration instead.

**Comment by semlanik** [comment]
@unC0Rr the idea is to provide deeper integration with Qt, std::duration is not part of Qt metaobject system. Overall not sure how useful will be such convertors, but hope it will be easier to use QTime instead of google.protobuf.Duration in some cases.

**Comment by unC0Rr** [comment]
Yeah, I understand the idea, but Qt itself has no means to express durations, and probably the only place you can find them in Qt is timer periods, which accept std::duration.
